filament/backend/src/vulkan/VulkanHandles.h

```diff
@@ -308,6 +308,17 @@ struct VulkanRenderTarget : private HwRenderTarget, fvkmemory::Resource {
     // Creates a special "default" render target (i.e. associated with the swap chain)
     explicit VulkanRenderTarget();
 
+    VulkanRenderTarget(VulkanRenderTarget&& target) {
+        (*this) = std::move(target);
+    }
+
+    VulkanRenderTarget& operator=(VulkanRenderTarget&& target) {
+        std::swap(mOffscreen, target.mOffscreen);
+        std::swap(mProtected, target.mProtected);
+        std::swap(mInfo, target.mInfo);
+        return *this;
+    }
```

Comment:
I would prefer that we create a swap() method that both ctor&& and operator=&& call.